README.md

```diff
@@ -58,72 +57,107 @@ Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP
 4. Change group ownership of repo (needs to be redone when a new file occurs in the repo):
```

Comment:
In Step 2 above, it is necessary to have this repository cloned locally in providing the `<path/to/repository>` . That being the case, I suggest we move the `Getting FIM source code` section above the  `Installation` section.